### ISSUE TITLE
DbError {info:"SqliteFailure{Error{code:Unknown,extended_code:1}, Some("near\"select\":syntax error")),kind: Other}

### ISSUE THREAD

**Issue by omar-beltagy** [ankipalace/advanced-browser#143]
DbError {info:"SqliteFailure{Error{code:Unknown,extended_code:1}, Some(\"near\\\"select\\\":syntax error\")),kind: Other} 
always get  this and stop whole browsing until disabling it

**Comment by ijgnd** [comment]
a bug report is much more useful if you post the anki version you use and if you've disabled all other add-ons and have restarted anki. There should be more than 1200 add-ons and bad interactions between add-ons are one of the most common problems, see also https://faqs.ankiweb.net/when-problems-occur.html

Maybe this makes the add-on mostly useable again: https://github.com/ankipalace/advanced-browser/issues/139#issuecomment-1040110225

**Comment by ijgnd** [comment]
The error message shown here matches exactly the error message posted as a negative review on ankiweb on 9/23/2022. This error message has slightly different spacing and one quote sign less than all other DbErrors isses we got in the past. 

What's unusual about this error is that there's a full traceback and not just only a one-line message about the DBError. Maybe this one is actually different.

The full error message apparently is:

```
Caught exception:
Traceback (most recent call last):
File "aqt.webview", line 568, in handler
File "aqt.editor", line 561, in <lambda>
File "aqt.utils", line 1018, in <lambda>
File "aqt.browser.browser", line 528, in on_table_state_changed
File "aqt.browser.table.table", line 193, in toggle_state
File "aqt.browser.table.model", line 243, in toggle_state
File "aqt.browser.table.model", line 260, in search
File "aqt.browser.table.state", line 156, in find_items
File "anki.collection", line 630, in find_cards
File "anki._backend.generated", line 1466, in search_cards
File "anki._backend", line 146, in _run_command
anki.errors.DBError: DbError { info: "SqliteFailure(Error { code: Unknown, extended_code: 1 }, Some(\"near \\\"select\\\": syntax error\"))", kind: Other }
```

**Comment by ijgnd** [comment]
@omar-beltagy: Let me be a bit more clear than four days ago: Your bug report is virtually useless unless you provide additional info.

Since no one is paid to make this add-on bug-free this bug won't be fixed in the coming years without a step-by-step guide on how to reproduce it because you're the first to report this even though this add-on in widely used.

1. yesterday a new version of ankibrowser was released. update your add-ons from ankiweb, restart anki. then try again. Does this fix your error?
2. If not: Disable all other add-ons, restart Anki and try again: Does the error still occur?
3. Reset the add-on config for Advanced Browser, restart Anki and try again: Does the error still occur?
4. If yes: Describe exactly what you did to get this error. Does this always happen when you open the browser or only in certain circumstances? Most importantly: which columns in the browser table are active and what is your sort column? Does my suggestion in the last sentence from my answer 4 days ago help?

**Comment by ijgnd** [comment]
probably same as #139 and fixed with #144. If not reopen this issue, see https://github.com/ankipalace/advanced-browser/issues/143#issuecomment-1259273017
